html-forms/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <div>
+        Email
+      </div>
+      <div>
+        <label for="email">
+          <input type="text" name="email">
+        </label>
+      </div>
+      <div>
+        <label for="coding-cool">
+          Why is coding cool?
+        </label>
+      </div>
+      <div>
+        <select name="coding">
+          <option value="select">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <p>Favorite Language (choose one)</p>
+      </div>
+      <div>
+        <input type="radio" name="language" value="html">
+        <label for="html">HTML</label>
+        <input type="radio" name="language" value="css">
+        <label for="css">CSS</label>
+        <input type="radio" name="language" value="javascript">
+        <label for="javascript">JavaScript</label>
+      </div>
+      <div>
+        <p>Would you like to sign up for more information?</p>
+      </div>
+      <div>
+        <label for="sign-up">
+          <input type="checkbox">
+          Sign me up!
+        </label>
+      </div>
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <div>
+        <textarea name="comments" cols="30" rows="10"></textarea>
```

Comment:
Include an `id` for `textarea` that corresponds to the `label`s `for` attribute.